src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutValenceCheckTest.java

```diff
@@ -0,0 +1,91 @@
+package org.openstreetmap.atlas.checks.validation.linear.edges;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Tests for {@link RoundaboutValenceCheck}
+ *
+ * @author savannahostrowski
+ */
+public class RoundaboutValenceCheckTest
+{
+
+    private final RoundaboutValenceCheck check = new RoundaboutValenceCheck(
```

Comment:
Is this used to test against? If not, you can use it for a test with different configuration values which would be nice to have.

Reply:
Whoops, forgot to actually use that configuration 😺 . Added some unit tests using an increased number of maximum connections. Let me know if you had something different in mind.